src/styles/forms.scss

```diff
@@ -0,0 +1,49 @@
+@import 'variables';
+
+[type="submit"] {
+    background-color: color(primary);
+    color: white;
+    font-size: 150%;
+}
+
+fieldset {
+    &.invalid,
+    input:invalid,
+    select:invalid {
+        box-shadow: 0 0 5px color(danger);
+    }
+
+    &.labeled-inputs {
+        border: 0;
+        border-spacing: 0.5rem;
+        display: table;
+
+        > label {
+            display: table-row;
+
+            > span,
+            > input {
+                display: table-cell;
+            }
+        }
+    }
+
+    &.labeled-buttons {
+        border: 0;
+
+        > label {
+            display: block;
+        }
+    }
+
+    > .validation-message {
+        display: none;
+    }
+
+    &.invalid > .validation-message {
+        background-color: white;
```

Comment:
Try to use variables for colors, even black and white.  `white` is `color(neutral-lightest)`.